web-server/algorand_core/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+const HASH_LEN: usize = 32;
```

Comment:
For these crates, please add `#![no_std]` to the top of the `lib.rs` files. This tells rust to build this without attempting to link to `std` and if it fails it will give a useful error.